modules/related-posts/jetpack-related-posts.php

```diff
@@ -1407,10 +1386,8 @@ protected function _get_related_posts( $post_id, $size, array $filters ) {
 				$filters
 			)
 		);
-
 		/** This filter is already documented in modules/related-posts/related-posts.php */
 		$hits = apply_filters( 'jetpack_relatedposts_filter_hits', $hits, $post_id );
-
```

Comment:
In general, I think all line breaks should stay where they were before, unless it's an awkward double line break that doesn't make sense. If there's a line break, it's because someone saw that they could read the code block or whatever more clearly. Like a line break after a paragraph. Removing that line break would get in their way.
Also, WP PHP coding standards doesn't say anything about line breaks. Plus, if you check new work like the REST API, there are line breaks here and there like:
```php
}

// Wrap the response in an envelope if asked for.
if ( isset( $_GET['_envelope'] ) ) {
	$result = $this->envelope_response( $result, isset( $_GET['_embed'] ) );
}

// Send extra data from response objects.
$headers = $result->get_headers();
$this->send_headers( $headers );

$code = $result->get_status();
$this->set_status( $code );
```

That said, thanks for all the curly brackets, spacing and alignment fixes!